### PR TITLE
uptick microsoft ads 

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_microsoft_ads_source/issues/30 by way of including https://github.com/fivetran/dbt_microsoft_ads_source/pull/31 

**This PR will result in the following new package version:**

v0.1.8.0

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [x] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
    
- [ ] docs were regenerated (unless this PR does not include any code or yml updates)
- [ ] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

#### ❗ Special Updates for Ad Reporting ❗
To reduce integration testing time, not all models should be enabled in the `run_models.sh` vars. Update the variables after `dbt run` and `dbt test` to set:
- [x] this PR's package to `true`
- [x] Google Ads and Facebook Ads to `true` (if not already)
- [x] All other packages to `false`

### Detailed Validation
Please share any and all of your validation steps:

see validations in https://github.com/fivetran/dbt_microsoft_ads/pull/32

### If you had to summarize this PR in an emoji, which would it be?
 
🤹

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -14,8 +14,10 @@ packages:
   - package: fivetran/linkedin
     version: [">=0.8.0", "<0.9.0"]
 
-  - package: fivetran/microsoft_ads
-    version: [">=0.7.0", "<0.8.0"]
+  # - package: fivetran/microsoft_ads
+  #   version: [">=0.8.0", "<0.9.0"]
+  - git: https://github.com/fivetran/dbt_microsoft_ads.git
+    revision: bugfix/campaign-report-add-budget
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:20]: Reminder to swap before release

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# dbt_ad_reporting v1.8.0
+Tiny release here!
+
+[PR #103](https://github.com/fivetran/dbt_ad_reporting/pull/103) just upticks the `microsoft_ads` dependency following the a major release. Refer to the Microsoft Ads v0.8.0 [release notes](https://github.com/fivetran/dbt_microsoft_ads/releases/tag/v0.8.0) for what exact updates have been included.
+
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:5]: Can we actually still have this be part of a sub section, just to keep the same structure as the other releases. For example something like the following:
```suggestion

[PR #103](https://github.com/fivetran/dbt_ad_reporting/pull/103) includes the following update:

## Dependency Updates
- Tiny update which upticks the `microsoft_ads` dependency following a recent major release. Refer to the Microsoft Ads v0.8.0 [release notes](https://github.com/fivetran/dbt_microsoft_ads/releases/tag/v0.8.0) for what exact updates have been included.
```

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-jamie thanks for opening this PR! This looks good to me, just one small request to reword the CHANGELOG a bit.

**Review by fivetran-catfritz — APPROVED** [review]
lgtm!

**Pull request merged**